### PR TITLE
README.md changes

### PR DESCRIPTION
**Pull request by hm90121**
-- add blobber build scripts for apple silicon devices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,13 @@ block_worker: https://demo.zus.network/dns
 Note: Run all scripts as sudo. 
 This would take few minutes.
 
+Sometimes in Apple Silicon devices (m1/m2 macbooks), buildx will not work for build scripts. To force a regular blobber build, run the following instead of the build commands mentioned above:
+```
+DOCKER_BUILD=build ./docker.local/bin/build.base.sh
+DOCKER_BUILD=build ./docker.local/bin/build.blobber.sh
+DOCKER_BUILD=build ./docker.local/bin/build.validator.sh.
+```
+
```
Comment (devyetii) [inline README.md:102]: Looks fine but please remove line 104 down starting with "For Mac with Apple M1 ..."

**Review by devyetii — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -88,6 +88,13 @@ block_worker: https://demo.zus.network/dns
 Note: Run all scripts as sudo. 
 This would take few minutes.
 
+Sometimes in Apple Silicon devices (m1/m2 macbooks), buildx will not work for build scripts. To force a regular blobber build, run the following instead of the build commands mentioned above:
```
Comment (Kishan-Dhakan) [inline README.md:96]: Please check and remove/clean up other references to M1 or apple devices

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Instead of a Note, please add a heading so that the following structure is used:

## Building and Starting the Nodes
### Building on standard hardware
... content
### Building on apple silicon (M1/M2 chips)
... content

**Review by Kishan-Dhakan — APPROVED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  | staging  |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Pull request merged**